### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by YuriyBilskiy**
[DEMO LINK](https://yuriybilskiy.github.io/react_tabs-js/)

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -9,32 +10,20 @@ export const tabs = [
...
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
+export const App = () => {
+  const [value, setValue] = useState({
+    id: 'tab-1', title: 'Tab 1', content: 'Some text 1',
+  });
```
Comment (Anastasiia-Svintsova) [inline src/App.jsx:16]: Rename value to be more descriptive.
image
```suggestion
  const [value, setValue] = useState(tabs[0].id);
```

File: src/App.jsx [hunk 2]
```diff
@@ -9,32 +10,20 @@ export const tabs = [
...
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-          </li>
+  return (
+    <div className="section">
+      <h1 className="title">
+        Selected tab is
+        {' '}
+        {value.title}
```
Comment (Anastasiia-Svintsova) [inline src/App.jsx:23]: Consider using [Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) for better readability

Comment (Anastasiia-Svintsova) [inline src/App.jsx:23]: Use destructuring for the objects for better readability

File: src/components/Tabs/Tabs.jsx [hunk 3]
```diff
@@ -1 +1,27 @@
-export const Tabs = () => {};
+export const Tabs = ({ value, setValue, tabs }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(x => (
```
Comment (Anastasiia-Svintsova) [inline src/components/Tabs/Tabs.jsx:5]: What does x mean? 
```suggestion
        {tabs.map(tab => (
```

File: src/components/Tabs/Tabs.jsx [hunk 4]
```diff
@@ -1 +1,27 @@
...
+            >
+              {x.title}
+            </a>
+          </li>
+        ))}
+      </ul>
+    </div>
+
+    <div className="block" data-cy="TabContent">
+      {value.content}
```
Comment (Anastasiia-Svintsova) [inline src/components/Tabs/Tabs.jsx:24]: Destructuring

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]
Keep going! 

Try to pass tests 
image

File: src/App.jsx [hunk 5]
```diff
@@ -9,32 +10,19 @@ export const tabs = [
...
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
+export const App = () => {
+  const [value, setValue] = useState(tabs[0].id);
+  const { title } = value;
```
Comment (Viktor-Kost) [inline src/App.jsx:15]: 1. it'd more descriptive to name selectedTabId
2. no needs to use destruction if you get only one value from 
```suggestion
  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
```

File: src/App.jsx [hunk 6]
```diff
@@ -9,32 +10,19 @@ export const tabs = [
...
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-          </li>
+  return (
+    <div className="section">
+      <h1 className="title">
+        Selected tab is
+        {' '}
+        {title}
```
Comment (Viktor-Kost) [inline src/App.jsx:22]:
```suggestion
        {selectedTabId}
```

File: src/App.jsx [hunk 7]
```diff
@@ -9,32 +10,19 @@ export const tabs = [
...
-          <li data-cy="Tab">
-            <a href="#tab-3" data-cy="TabLink">Tab 3</a>
-          </li>
-        </ul>
-      </div>
-
-      <div className="block" data-cy="TabContent">
-        Some text 1
-      </div>
+      <Tabs value={value} setValue={setValue} tabs={tabs} />
```
Comment (Viktor-Kost) [inline src/App.jsx:25]:
```suggestion
      <Tabs 
        selectedTabId={selectedTabId} 
        setSelectedTabId={setSelectedTabId} 
        tabs={tabs} 
      />
```

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
GJ!
Just pay attention at previous mentor comment
https://github.com/mate-academy/react_tabs-js/pull/239#discussion_r1308670926

**Pull request closed without merging**